src/api/app/views/event_mailer/comment_for_project.html.haml

```diff
@@ -1,5 +1,7 @@
+- url = url_for(project: event['project'], controller: 'webui/project', action: :show, only_path: false, host: @host)
+
 %p
-  Visit #{link_to(*[url_for(project: event['project'], controller: 'webui/project', action: :show, only_path: false, host: @host)]*2)}
+  Visit #{link_to(*[url] * 2)}
```

Comment:
that's a trick that actually do not make the code better. That's just to make the linter happy.

Reply:
It's true... I'm guilty :smile:

Reply:
you are not guilty. The LineLength cop is :)

Reply:
I found a better solution, please have a look.